src/app.rb

```diff
@@ -0,0 +1,168 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'base_decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    create_initial_data
+  end
+
+  # Create some initial data
+  def create_initial_data
+    # Create students
+    student1 = Student.new(15, name: 'John', parent_permission: true)
+    student2 = Student.new(16, name: 'Alice', parent_permission: true)
+
+    # Create a teacher
+    teacher1 = Teacher.new(35, name: 'Mr. Smith', specialization: 'Math')
+
+    # Create books
+    book1 = Book.new('Introduction to Ruby', 'John Rubyist')
+    book2 = Book.new('Ruby on Rails Guide', 'Ruby Rails')
+
+    # Create rentals
+    rental1 = Rental.new('2023-10-12', book1, student1)
+    rental2 = Rental.new('2023-10-10', book2, student2)
+
+    # Add the created objects to your data
+    @people.push(student1, student2, teacher1)
+    @books.push(book1, book2)
+    @rentals.push(rental1, rental2)
+  end
+
+  # List of books...
+  def list_books
+    puts(@books.map { |book| "Title: \"#{book.title}\", Author \"#{book.author}\"" })
+  end
+
+  # List of people...
+  def list_people
+    puts(@people.map { |person| "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" })
+  end
```

Comment:
- [ ] Please only display the added users once the app is run as shown in the [reference video](https://www.youtube.com/watch?v=vkkgrhD6aXQ). you are not required to have already created teacher or student :+1: :sparkles: 
![3](https://github.com/Kaiserabbas/school_library/assets/89055502/8899582c-a55d-44ab-a919-3c7b94ff137d)

Reply:
[Update the code by removing initial data supplied to the app](https://github.com/Kaiserabbas/school_library/pull/4/commits/5fb09cf98f3d48509812bb27f0ce5dbc12be1d40)